Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, host, options, gitClass=Git, bugzillaClass=Bugzilla, WPTGitHu
 
         self._WPTGitHubClass = WPTGitHubClass
         self._gitClass = gitClass  # FIXME: Move git operations to webkitscmpy
-        self._repository = repository or local.Git(os.path.abspath(os.getcwd()))
```

Comment:
Can we not just use WebKitFinder? Do something like https://github.com/WebKit/WebKit/blob/627fbaff8094b58ecbb5817511f77d5b91d96e2f/Tools/Scripts/webkitpy/common/net/bugzilla/test_expectation_updater.py#L232-L235?